src/site/stages/build/process-cms-exports/helpers.js

```diff
@@ -41,6 +41,33 @@ function getContentModelType(entity) {
   return [entity.baseType, subType].filter(x => x).join('-');
 }
 
+/**
+ * Add common properties to the entity. Does not mutate `entity`; instead,
+ * returns a new object.
```

Comment:
Is there an advantage here to not modifying the entity directly?

Reply:
Yeah, I was concerned about the memory it'd use to create new objects for every single entity. Tens of thousands of times. Then you consider the added time for creation of those objects, and the added time for garbage collection, and a seemingly simple thing becomes kind of a pain.

Reply:
To be fair, I didn't run any tests. It was all assumption.